### ISSUE TITLE
Create installable artifacts for Blazor hybrid desktop apps for Windows, macOS, and Linux

### ISSUE THREAD

**Issue by danroth27** [dotnet/aspnetcore#27234]
### Summary

As a [developer], I want to [create installable artifacts for my Blazor hybrid desktop app] so that [I can distribute my app].”

> Note: This is a proposed experience for .NET 6 and not yet committed.

**Comment by mkArtakMSFT** [comment]
@rconard I don't think this should be tracked in this repo, as it's a broader work. Is there any other issue tracking this we can dupe this with?

**Comment by danroth27** [comment]
Scoping out Linux for .NET 6. We expect Linux support to come later.

**Comment by SommerEngineering** [comment]
@danroth27 Can we get another issue i.e. user story for Linux then? The related epic #27217 links to this issue with the title: "Create installable artifacts for Blazor hybrid desktop apps for Windows, macOS, **and Linux**". When Linux was scoped out, we Linux users would need a new issue to follow :wink:

**Comment by tmds** [comment]
> Scoping out Linux for .NET 6. We expect Linux support to come later.

@danroth27 why? are there some specific challenges to make this work on Linux? Anything community could help with?

**Comment by danroth27** [comment]
It's really a scoping decision for .NET MAUI in .NET 6. .NET MAUI is an evolution of Xamarin Forms, which currently has official support for Android, iOS, and UWP. .NET MAUI will add Windows desktop support with WinUI, and macOS support. Linux just didn't fit for .NET 6, but we absolutely still plan to do it. Blazor will run wherever .NET MAUI runs.

Adding @davidortinau from the .NET MAUI team in case he wants to add anything.

**Comment by pjmagee** [comment]
Would this also include things like Auto update features?
